genesyscloud/resource_genesyscloud_webdeployments_configuration.go

```diff
@@ -119,6 +120,12 @@ var (
 				Optional:    true,
 				Computed:    true,
 			},
+			"channels": {
+				Description: "List of channels through which cobrowse is available (for now only Webmessaging and Voice)",
+				Type:        schema.TypeList,
+				Optional:    true,
+				Elem:        &schema.Schema{Type: schema.TypeString},
```

Comment:
You should add a validation function here; `ValidateFunc: validation.StringInSlice(...`

Reply:
Here's an example of how you can validate the elements within the list: [https://github.com/MyPureCloud/terraform-provider-genesyscloud/blob/main/genesyscloud/outbound/resource_genesyscloud_outbound_dnclist.go#L92](https://github.com/MyPureCloud/terraform-provider-genesyscloud/blob/main/genesyscloud/outbound/resource_genesyscloud_outbound_dnclist.go#L92)

Reply:
Thanks, updated it accordingly.